src/schema/objects/metadata.yaml

```
@@ -3366,6 +3366,20 @@ SubjectArtefactDescription:
     If this field is set to `"n/a"`, it will be interpreted as absence of major
     source of artifacts except cardiac and blinks.
   type: string
+TablePosition:
+  name: TablePosition
+  display_name: Table Position
+  description: |
+    The table position, relative to an implementation-specific reference point,
+    often the isocenter. Values must be an array of three distances in
+    millimeters, respectively right, anterior and superior (RAS+) of the image
+    affine.
```

Apologies, I think I gave you some slightly unclear text. I think this is clearer, but please feel free to use an alternative that makes sense to you.

```suggestion
    millimeters, respectively right, anterior and superior (RAS+) of the origin
    defined by the image affine.
```

---

Good point! I reworded it a little more